gcp/modules/gcp-stackdriver-export/main.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "project_id" {}
+variable "serviceaccount_key" {}
+variable "exports" {
+  default = {
+    # Captures interactions with many Google products, including GCE, GS, IAM,
+    # and more.
+    #
+    # TODO: This is a lot less noisy if we add "AND NOT operation.producer=k8s.io".
+    "cloudaudit-activity" = "logName=projects/__var.project_id__/logs/cloudaudit.googleapis.com%2Factivity"
+
+    # Captures Google Storage and KMS events
+    "cloudaudit-data-access" = "logName=projects/__var.project_id__/logs/cloudaudit.googleapis.com%2Fdata_access"
+
+    # Captures events for products within GCE, such as Snapshots, Instance
+    # Groups, Firewalls, etc.
+    "compute-activity" = "logName=projects/__var.project_id__/logs/compute.googleapis.com%2Factivity_log"
+  }
```

Comment:
Seems like we have the `${var.project_id}` var here, is there a `__var.project_id__` need for the replacement bit?

Reply:
This discussion is continuing in https://github.com/gpii-ops/exekube/pull/12.